include/nccl_ofi_log.h

```diff
@@ -24,11 +24,14 @@ extern ncclDebugLogger_t ofi_log_function;
 	__PRETTY_FUNCTION__, __LINE__, "NET/OFI " fmt,		\
 	##__VA_ARGS__)
 
+#if OFI_NCCL_TRACE
```

Comment:
This has the unfortunate side effect that it becomes significantly more difficult to ask customers to rerun with `NCCL_DEBUG=Trace` to debug plugin initialization issues. We'd have to ask them to recompile the plugin, which is especially difficult when using containers.

Reply:
So we should do a bunch of unnecessary string parsing in the critical path instead?

Reply:
Yeah probably not, especially since we're matching NCCL's behavior with this PR.